lmeval.py

```diff
@@ -125,7 +124,7 @@ def main():
         num_fewshot=args.num_fewshot,
         batch_size=args.batch_size,
         device=args.device,
-        no_cache=args.no_cache,
+        no_cache=True,
```

Comment:
no_cache is recommended to be set true in the tests. Also no_cache=False is not compatible with evaluating pre-loaded models (incorrect code in lm-eval). Now it is set to True by default for convenience, option removed.